nifi-extension-bundles/nifi-box-bundle/nifi-box-processors/src/main/java/org/apache/nifi/processors/box/GetBoxFileCollaborators.java

```diff
@@ -79,6 +80,25 @@ public class GetBoxFileCollaborators extends AbstractProcessor {
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor ROLES = new PropertyDescriptor.Builder()
+            .name("Roles")
+            .description("A comma-separated list of collaboration roles to retrieve. Available roles: editor, viewer, previewer, " +
+                    "uploader, previewer uploader, viewer uploader, co-owner, owner")
+            .required(true)
+            .defaultValue("editor")
+            .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .build();
+
+    public static final PropertyDescriptor STATUSES = new PropertyDescriptor.Builder()
+            .name("Statuses")
+            .description("A comma-separated list of collaboration statuses to retrieve. Available statuses: accepted, pending, rejected")
+            .required(true)
+            .defaultValue("accepted")
+            .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .build();
+
```

Comment:
As a follow up of my other comment, can we make those two new properties as optional with no default value and create the new attributes only if someone provides a value for those properties?